rust/phoenix-sdk-core/src/sdk_client_core.rs

```diff
@@ -101,24 +103,33 @@ impl SDKClientCore {
     /// Converts base units to base lots. For example if the base currency was a Widget and you wanted to
     /// convert 3 Widgets to base lots you would call sdk.base_unit_to_base_lots(3.0). This would return
     /// the number of base lots that would be equivalent to 3 Widgets.
-    pub fn base_units_to_base_lots(&self, base_units: f64) -> u64 {
-        (base_units * self.base_multiplier as f64 / self.base_lot_size as f64) as u64
+    pub fn raw_base_units_to_base_lots(&self, raw_base_units: f64) -> u64 {
+        // Convert to Phoenix BaseUnits
+        let phoenix_base_units = raw_base_units / self.raw_base_units_to_base_units as f64;
+        (phoenix_base_units * (self.num_base_lots_per_base_unit as f64)).floor() as u64
+    }
+
+    /// The same function as raw_base_units_to_base_lots, but rounds up instead of down.
+    pub fn raw_base_units_to_base_lots_rounded_up(&self, raw_base_units: f64) -> u64 {
+        // Convert to Phoenix BaseUnits
+        let phoenix_base_units = raw_base_units / self.raw_base_units_to_base_units as f64;
+        (phoenix_base_units * (self.num_base_lots_per_base_unit as f64)).ceil() as u64
     }
 
     /// RECOMMENDED:
-    /// Converts base amount to base lots. For example if the base currency was a Widget with 9 decimals and you wanted to
+    /// Converts base atoms to base lots. For example if the base currency was a Widget with 9 decimals, where 1 atom is 1e-9 of one Widget and you wanted to
     /// convert 3 Widgets to base lots you would call sdk.base_amount_to_base_lots(3_000_000_000). This would return
-    /// the number of base lots that would be equivalent to 3 Widgets.
-    pub fn base_amount_to_base_lots(&self, base_amount: u64) -> u64 {
-        base_amount / self.base_lot_size
+    /// the number of base lots that would be equivalent to 3 Widgets or 3 * 1e9 Widget atoms.
+    pub fn base_atoms_to_base_lots(&self, base_atoms: u64) -> u64 {
+        base_atoms / self.base_lot_size //Lot size is the number of atoms in a lot
```

Comment:
nit: space after // (and the same below)

Reply:
Changed to reflect in latest commit